uwsift/view/layer_details.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, document, parent=None):
         self.clims_text.setSizePolicy(QSizePolicy.Fixed, QSizePolicy.Fixed)
         self.cmap_vis = QNoScrollWebView()
         self.cmap_vis.setFixedSize(min_width, 30)
-        self.cmap_vis.page().mainFrame().setScrollBarPolicy(Qt.Vertical, Qt.ScrollBarAlwaysOff)
+        self.cmap_vis.page().runJavaScript('document.body.style.overflow = "hidden";')
```

Comment:
Is `QNoScrollWebView` used any where else in SIFT? Is it possible to add CSS/styling information to the HTML that gets added to the web view?

Reply:
No, it's not used anywhere else in SIFT, but I could add something like:

`<style type="text/css">
body {
    overflow:hidden;
}
</style>`

to the HTML on lines 120 and 125 instead?

Reply:
I guess it might be better the way you have it since this javascript should only run once, right?

Reply:
Yeah, I think it should only run once.